### DISCUSSION TITLE
Rocket_ws on crates.io

### DISCUSSION THREAD

**Discussion by DraconovRyan04** [rwf2/Rocket#2609] [Questions]
Why didn't you upload rocket_ws on crates.io? I would like to try to use it in my project to make TaskManager reloading in real time. If there are any articles on how to make Rocket friends with Web Sockets, I would be glad to read them.

**Comment by the10thWiz** [comment]
`rocket_ws` is too new to be on crates.io at this point. It will likely be uploaded with the next release candidate and/or the 0.5 release. For now, you can use a git dependency to use the pre-release version.

**Comment by DraconovRyan04** [comment]
Is there any way to somehow send a message to all users when you receive a message from one of them?
